org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/internal/DebugSession.java

```diff
@@ -20,13 +20,16 @@
 import com.sun.jdi.ThreadReference;
 import com.sun.jdi.VirtualMachine;
 import com.sun.jdi.request.EventRequest;
+import com.sun.jdi.request.EventRequestManager;
 
 public class DebugSession implements IDebugSession {
     private VirtualMachine vm;
+    private EventRequestManager eventRequestManager;
     private EventHub eventHub = new EventHub();
 
     public DebugSession(VirtualMachine virtualMachine) {
         vm = virtualMachine;
+        eventRequestManager = vm.eventRequestManager();
```

Comment:
No need to cache eventRequestManager in local variable. Invoke vm.eventRequestManager to get it directly.